### DISCUSSION TITLE
Uno.UI.Xaml and DependencyProperty

### DISCUSSION THREAD

**Discussion by AghaKhan** [unoplatform/uno#7216] [Q&A]
### Current behavior

I have a Library where I wanted to create some Dependency Properties but I am unable to find Uno.UI.Xaml. Obviously by default Uno.UI is installed. As I understand you provide DependencyProperty Generator, but where I can find it? 
I would like to create 
public static readonly DependencyProperty LocationProperty = DependencyProperty.Register("Location", typeof(Point), typeof(HexagonCrl));
   
and 
public static readonly DependencyProperty DataProperty = DependencyProperty.Register("Data", typeof(ObservableCollection<int>), typeof(HexagonCrl), new PropertyMetadata(null, NutNumbersChanged));

Any help will be very much appreciated.

**Comment by jeromelaban** [comment]
I've convert the issue to a discussion, so that we can better understand what you're trying to achieve.

Could you provide a sample app of what you're trying to do? 

In practice, there should be no difference with the code that you write when running in the UWP head.
